app/src/test/java/com/appunite/loudius/domain/PullRequestRepositoryImpTest.kt

```diff
@@ -46,35 +50,48 @@ class PullRequestRepositoryImpTest {
     inner class GetReviewsFunctionTest {
 
         @Test
-        fun `GIVEN correct values WHEN get reviews THEN return result with reviews excluding ones from current user`() =
+        fun `GIVEN correct pull request data WHEN getting reviews THEN return result with reviews excluding ones from current user`() =
             runTest {
-                val actual = repository.getReviews("example", "example", "correctPullRequestNumber")
+                val pullRequestNumber = "correctPullRequestNumber"
```

Comment:
question: why do you extract this variable: `val pullRequestNumber = "correctPullRequestNumber"`  ithink it makes harder to understand this test.

Reply:
I did it because I thought an explicit 'GIVEN' part would more clearly show what is the difference between the tests